src/utils/files.py

```
@@ -223,3 +225,20 @@ async def process_file_uploads_for_messages(
+    """
+    Load a prompt template from a file and return a Jinja2 Template object.
+    """
+    try:
+        env = Environment(loader=FileSystemLoader(prompt_path))
+        template = env.get_template(template_filename)
+        return template
+    except TemplateNotFound:
+        return None
+    except Exception as e:
+        logger.warning("Error loading prompt template: %s", e)
+        raise e
```

Ruff (0.14.2)

Use `raise` without specifying exception name

Remove exception name

(TRY201)